macros/CSSSyntax.ejs

```diff
@@ -43,13 +37,8 @@ let externallyDescribedTypesData = {
   }
 };
 let data = {
-    properties: await MDN.fetchJSONResource(propertiesUrl),
-    atRules: await MDN.fetchJSONResource(atRulesUrl),
-    selectors: await MDN.fetchJSONResource(selectorsUrl),
-    types: await MDN.fetchJSONResource(typesUrl),
-    syntaxes: await MDN.fetchJSONResource(syntaxesUrl),
-    units: await MDN.fetchJSONResource(unitsUrl),
-    externallyDescribedTypes: externallyDescribedTypesData
+    ...require('mdn-data/css'),
```

Comment:
I'd prefer to see a line up at the top `const cssdata = require('mdn-data/css')` and then here just do `...cssdata`.  This code is technically correct, but could be confusing, and it ends up hiding a require in the middle of the file rather than putting it up top.